src/SFA.DAS.AssessorService.Web.Staff/Startup.cs

```diff
@@ -118,7 +118,7 @@ private IServiceProvider ConfigureIoC(IServiceCollection services)
                     _.AssemblyContainingType(typeof(Startup));
                     _.WithDefaultConventions();
                 });
-                config.For<ITokenService>().Use<TokenService>();
+                config.For<ITokenService>().Use<TokenService>().Ctor<bool>("useSandbox").Is(false); // Always false unless we want to start integrating with the sandbox environment
```

Comment:
Changing the Web.Staff project here this should be change to Admin Service instead or as well as

Reply:
As it stands Admin Service won't need to know about the Sandbox. In this solution it needs to be aware as to not to confuse a developer. Effectively this will become legacy as soon as Admin service is live (and this project deleted)